src/MBCs.md

```diff
@@ -20,3 +20,10 @@ so, it might be nevertheless possible to use Double Speed during periods
 which use only code and data which is located in internal RAM. Despite the 
 above, a self-made MBC1-EPROM card appears to work stable and fine even in 
 Double Speed Mode.
+
+## MBC Unmapped RAM Bank Access
+
+On most of the MBC's in case an unmapped RAM bank is selected (which would be translate to an out of bounds RAM address by the MBC controller), 
+the MBC controller will simply wrap around the internal ram address and would access valid RAM address.
+
+The MBC internal address being accessed can be calculated using this formula: `((address - external_ram_start_address) * ram_bank_size) % max_external_ram_size`.
```

Comment:
I'd have expected `(active_ram_bank % nb_ram_banks) * 0x1000 + (address - 0xA000)`. Am I missing something?

Reply:
You are correct, I forgot to add the active ram bank to the formula.

I wrote the formula differently but they are supposed to be equivalent (unless I made a mistake again), let me know which way you prefer it.